src/core/js/views/questionView.js

```diff
@@ -307,7 +319,7 @@ define([
     refresh() {
       this.model.set('_buttonState', this.model.getButtonState());
 
-      if (this.model.get('_canShowMarking') && this.model.get('_isInteractionComplete') && this.model.get('_isSubmitted')) {
+      if (this.canShowMarking() && this.model.get('_isInteractionComplete') && this.model.get('_isSubmitted')) {
```

Comment:
Brackets can be removed as not a function any more
```suggestion
      if (this.canShowMarking && this.model.get('_isInteractionComplete') && this.model.get('_isSubmitted')) {
```